apps/cli-daemon/src/app/workspace/workspace.controller.ts

```diff
@@ -102,4 +115,24 @@ export class WorkspaceController {
 
     return target;
   }
+
+  private ngNewArgsFromDto(dto: WorkspaceCreateDto): string[] {
+    const { name, options } = dto;
```

Comment:
if you don't need the actual dto just destruct it in the params

Reply:
👍